diff --git a/pyspglib/_spglib.c b/pyspglib/_spglib.c
index fb7bc85..8b9c8d2 100644
--- a/pyspglib/_spglib.c
+++ b/pyspglib/_spglib.c
@@ -183,7 +183,7 @@ static PyObject * get_version(PyObject *self, PyObject *args)
 
   array = PyList_New(3);
   for (i = 0; i < 3; i++) {
-    PyList_SetItem(array, i, PyInt_FromLong((long)version[i]));
+    PyList_SetItem(array, i, PyLong_FromLong((long)version[i]));
   }
 
   return array;
-- 
2.6.4

